source/introduction.rst

```diff
@@ -105,74 +144,86 @@ Object oriented programming
 
 We can see our world as containing different types of objects that we can classify according to common attributes and behaviours.
 
-For example in a classroom there many objects that are instances of the type Chair and many other objects that are instances of the type Person.
+For example in a classroom there many objects that are instances of the type `Chair` and many other objects that are instances of the type `Person`.
 
 Objects have:
 
 * Attributes - Chairs have four legs, Persons have two.
 * Behaviours - Persons can walk. Persons can move chairs.
 
 Objects can interact with other objects of different types. An object of type Person can
-can sit on an object of type Chair. 
+can sit on an object of type Chair.
 
 This is the essence of object oriented thinking. It is about using programmatic 
 objects to model a domain of interest to a programmer.
 
 This style of programming provides a clear, simple, and consistent model of computation
 that maps well to our intuitions about the world.
 
-Python
+More about Python
 ======
 
-Python is a simple to learn yet fully featured, high-level, object oriented programming language. It's popular both in academia, science and other industries. The concepts however will apply to most other object oriented languages. 
+Python is a simple to learn yet fully featured, high-level, object oriented programming language.
+
+Two terms to bear in mind:
 
-Two things to bear in mind:
+**A programming language**
+    A programming language is a formal computer language or constructed language designed to communicate instructions to a machine, particularly a computer. Programming languages can be used to create programs to control the behavior of a machine or to express algorithms.
 
-* A language - The textual instructions you type.
-* An interpreter - A program (called `python`) that reads and executes that language.
+In other words, the Python language is the textual instructions you type.
 
-Together we will learn the correct syntax and grammar of the Python language. 
+**An interpreter**
+    A program (called `python`) that reads and executes (runs) a programming language, line-by-line.
 
-When we ask the interpreter to execute it, it is interpreter that understands how to translate Python scripts into creating and manipulating objects according to your instructions.
+When we ask the interpreter to execute the Python we write, it understands how to translate Python scripts into the creation and manipulation of objects according to your instructions.
```

Comment:
It might be good to add something about translating from instructions that is easy for a human to read to instructions that a computer can understand